Automattic/pocket-casts-android
# [Project] Manage Downloaded Episodes - Snooze Low Storage Modal

## Description
- This Snooze the Low Storage modal for 7 days after the user taps on Maybe Later Button
- Context: p1730298732804909/1730291928.354409-slack-C07TL9YK9V1

Fixes #3152

## Testing Instructions

Use this [mock.patch](https://github.com/user-attachments/files/17602671/mock.patch) to mock low device storage and replace the snooze rule from **7 days to 10 seconds**

1. Download an episode
2. Go to Profile -> Storage & Data Use
3. ✅ Ensure the modal opens 
4. Tap on Maybe Later
5. Wait for more than 10 seconds
6. Open the Storage & Data Use again
7. ✅ Ensure you don't see the modal

## Screenshots or Screencast 
[Screen_recording_20241101_144020.webm](https://github.com/user-attachments/assets/050dcfa9-8de0-4a31-b8dc-c281510e0610)


## Checklist
 
#### I have tested any UI changes...
